packages/astro/src/prefetch/index.ts

```diff
@@ -0,0 +1,260 @@
+/*
+  NOTE: Do not add any dependencies or imports in this file so that it can load quickly in dev.
+*/
+
+// eslint-disable-next-line no-console
+const debug = import.meta.env.DEV ? console.debug : undefined;
```

Comment:
Out of curiosity, do we set `DEV` when using the dev server? Or are there specific astro options?

Reply:
It's from Vite.

Reply:
Yeah I know, I am asking when it's set :) during the `dev` phase only?

Reply:
Yup. But specifically if `NODE_ENV !== 'production'`, which should always be the case in dev. https://github.com/vitejs/vite/blob/ac964de760a455838aeeb5a1e197e5ab77731f59/packages/vite/src/node/config.ts#L767